lib/generators/suspenders/testing/minitest_generator.rb

```diff
@@ -0,0 +1,38 @@
+module Suspenders
+  module Generators
+    module Testing
+      class MinitestGenerator < Rails::Generators::Base
+        def add_gems
+          gem_group :test do
+            gem "webmock"
+          end
+
+          Bundler.with_unbundled_env { run "bundle install" }
+        end
+
+        def modify_test_helper
+          webmock_config = <<~RUBY
+
+            WebMock.disable_net_connect!(
+              allow_localhost: true,
+              allow: "chromedriver.storage.googleapis.com"
+            )
+          RUBY
+
+          insert_into_file "test/test_helper.rb", webmock_config
+
+          insert_into_file "test/test_helper.rb",
+            "\s\s\s\sinclude ActionView::Helpers::TranslationHelper\n" \
+            "\s\s\s\sinclude ActionMailer::TestCase::ClearTestDeliveries\n\n",
```

Comment:
This is to create parity with the [existing RSpec behavior](https://github.com/thoughtbot/suspenders/blob/55cb5c246fe53aea4cf53436c7c208b40ad13e85/lib/suspenders/generators/testing_generator.rb#L39-L41).

However, I don't know if this is the best way to do this?